keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -506,6 +503,36 @@ def _parse_backbone(backbone, include_rescaling, backbone_weights):
     return backbone
 
 
+def _parse_box_loss(loss):
+    if not isinstance(loss, str):
+        # support arbitrary callables
+        return loss
+
+    # case insensitive comparison
+    if loss.lower() == "smoothl1":
+        return keras_cv.losses.SmoothL1Loss(l1_cutoff=1.0, reduction="none")
+
+    raise ValueError(
```

Comment:
any particular reason that this cannot support Huber loss?

Reply:
Sure, why not!  added.